src/shared/utilities/messages.ts

```diff
@@ -215,3 +216,18 @@ export async function copyToClipboard(data: string, label?: string, env: Env = v
     vscode.window.setStatusBarMessage(addCodiconToString('clippy', message), 5000)
     getLogger().verbose('copied %s to clipboard: %O', label ?? '', data)
 }
+
+export async function showOnce<T>(
+    key: string,
+    fn: () => Promise<T>,
+    memento = globals.context.globalState
```

Comment:
If we "memoize" this in globalState it acts like "Don't show again", right? Which is different than e.g. `once()`. Need to think of a different name. Maybe `showOnceForever`